cvmfs/catalog_mgr_rw.cc

```diff
@@ -12,9 +12,12 @@
 #include <cassert>
 #include <cstdio>
 #include <cstdlib>
+#include <ctime>
```

Comment:
Which implies that each and every small fix like this one has to be carefully analysed before being fixed, "loosing" time in the process, and even more if these analysis were wrong. I doubt the time lost and effort put is worth having into account that the final status of the repository will be the same...